app/views/merchant/merchants/show.html.erb

```diff
@@ -4,5 +4,17 @@
   <h1><%= @merchant.name %></h1>
   <p>Email: <%= @merchant.email %></p>
   <p>Address: <%= @merchant.address %></p>
-  <p><%= @merchant.city %>, <%= @merchant.state %> <%= @merchant.zip %></p>
+  <p><%= @merchant.city %>, <%= @merchant.state %></p>
+  <p>Zip: <%= @merchant.zip %></p>
 </div>
+
+<h4>Pending Orders</h4>
+
+<% @merchant_orders.each do |order| %>
+  <div id="merchant-orders-<%= order.id %>">
+    <p><%= link_to "Order #{order.id}" %>, merchant_order_path(order)</p>
+    <p>Date Ordered: <%= Date.strptime(order.created_at.to_s) %></p>
+    <p>Total Quantity Ordered: <%= order.total_quantity_for_merchant(current_user) %></p>
```

Comment:
On line 17, it should be: order.total_quantity_for_merchant(@merchant).

In our view, we want to pass in the instance variable to our method.